python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -3972,7 +3972,22 @@ def callback(self, pre, post, node_map):
         
         return tvm.relay.nn.dense(act, dense.args[1])
 
+class ExpandMultipleDims(DFPatternCallback):
+    def __init__(self):
+        super().__init__()
+        self.act = wildcard()
+        self.pattern = is_op('expand_dims')(self.act)
 
+    def callback(self, pre, post, node_map):
+        act = node_map[self.act][0]
+        num_newaxis = int(post.attrs.num_newaxis)
```

Comment:
If we can't check for negative dims, please add assert to error out when we hit this case :))